pkg/manager/impl/execution_manager.go

```diff
@@ -568,13 +568,16 @@ func (m *ExecutionManager) launchSingleTaskExecution(
 		annotations = requestSpec.Annotations.Values
 	}
 
+	resolvedAuthRole := resolvePermissions(request, launchPlan)
+	resolvedSecurityCtx := resolveSecurityCtx(ctx, request, launchPlan, resolvedAuthRole)
 	executionParameters := workflowengineInterfaces.ExecutionParameters{
 		Inputs:              request.Inputs,
 		AcceptedAt:          requestedAt,
 		Labels:              labels,
 		Annotations:         annotations,
 		ExecutionConfig:     executionConfig,
-		Auth:                resolvePermissions(&request, launchPlan),
+		Auth:                resolvedAuthRole,
```

Comment:
So is it possible to just remove Auth field completely from ExecutionParameters? is this used anywhere that's not FlyteAdmin? if it only lives in flyteAdmin, then we can update the code to always use the SecurityContext and we make sure it's populated here.... right?

Reply:
+1 we should move away from passing Auth objects

Reply:
Make sense. Its not used outside admin. Have removed it from ExecutionParameters. It will remain in the ExecutionCreateRequest though until all clients migrate after which we can remove it completely along with the protos.

Reply:
Done

Reply:
one more thing to remove from flyteidl in flyte 1.0.0 😄